src/domain/ledger/index.types.d.ts

```diff
@@ -124,22 +124,22 @@ type AddColdStorageTxSendArgs = {
 type IntraledgerTxArgs = {
   senderWalletId: WalletId
   senderWalletCurrency: WalletCurrency
-  senderUsername: Username | null
+  senderUsername?: Username
   description: string
   sats?: Satoshis
   cents?: UsdCents
   recipientWalletId: WalletId
   recipientWalletCurrency: WalletCurrency
-  recipientUsername: Username | null
-  memoPayer: string | null
+  recipientUsername?: Username
```

Comment:
@dolcalmi Any thoughts on changing these two metadata value types from null to undefined?

Reply:
This is the place where transactions are brought from mongoose into our domain https://github.com/GaloyMoney/galoy/blob/21b76e1e261ea818f03d42a182b044a50397da8c/src/services/ledger/index.ts#L308

Reply:
The current [LedgerTransaction type](https://github.com/GaloyMoney/galoy/blob/e9a9686ee704bd6e7cc09834d09c266be9c58739/src/domain/ledger/index.types.d.ts#L24) actually already uses undefined, but I'm not sure how a null value persisted in the database becomes an undefined value.